.buildpacks

```diff
@@ -1,3 +1,3 @@
 https://github.com/unfold/heroku-buildpack-pnpm#76cf1fc
-https://github.com/heroku/heroku-buildpack-nginx#4cd76d0
+https://github.com/heroku/heroku-buildpack-nginx#v1.9
```

Comment:
```suggestion
https://github.com/heroku/heroku-buildpack-nginx#760407b
```

I think I'd prefer to keep pointing to a specific commit because git tags can be moved and this ensures that we will see an error if for some reason the tag changes.

Reply:
Given the repo is under the heroku org I'm hoping they won't pull such shenanigans, but it makes sense to stay consistent. I've updated the PR and will merge.